src/pages/patientView/PatientViewPage.tsx

```diff
@@ -38,10 +38,16 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
             );
         }
 
+        let samples = ["string", "TCGA-24-2024-01", "TCGA-24-2024-01 (deliberately elongated id)"];
```

Comment:
I see. In this case, we need to make sure that both genomic overview and `TumorColumnFormatter` receives the same "order of samples" from a single data source.